src/gui/torrentoptionsdialog.cpp

```diff
@@ -343,6 +414,20 @@ int TorrentOptionsDialog::getSeedingTime() const
     return m_ui->spinTimeLimit->value();
 }
 
+void TorrentOptionsDialog::handleTMMChanged()
+{
+    if (!m_ui->checkAutoTMM->isChecked())
+    {
+        m_ui->labelSavePath->setEnabled(true);
+        m_ui->savePath->setEnabled(true);
+    }
+    else
+    {
+        m_ui->labelSavePath->setEnabled(false);
+        m_ui->savePath->setEnabled(false);
```

Comment:
Shouldn't you also show automatically assigned save path here? Don't remember how "Add new torrent" dialog does it...

Reply:
All done.